xml/decoders/ESU_LokSound5.xml

```diff
@@ -20,12 +20,14 @@
     <!-- ver4 updates for F29-F31 -->
     <!-- ver5 disable Reset to Defaults -->
     <!-- ver6 add "LokSound 5 micro E24 DCC" and "LokSound 5 micro KATO" -->
+    <!-- ver6 renamed "LokSound 5 micro E24 DCC" to "LokSound 5 nano DCC" -->
```

Comment:
Should this be `ver7` instead of `ver6`?

Reply:
Thanks, fixed.